posts/2022-02-14-crates-io-snapshot-branches.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Crates.io Index Snapshot Branches Moving"
+author: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
```

Comment:
The blog doesn't render markdown for the author field.
This should be just the following:

```suggestion
author: The Crates.io Team
```

(that's used by at least one other blog post)